src/config/ConfigManager.cpp

```diff
@@ -683,6 +683,12 @@ std::string CConfigManager::getMainConfigPath() {
     if (!g_pCompositor->explicitConfigPath.empty())
         return g_pCompositor->explicitConfigPath;
 
+    if (const char *cfg = getenv("HYPRLAND_CONFIG"); cfg != NULL) {
+        return cfg;
+    }
+
+    Debug::log(INFO, "Seems as if HYPRLAND_CONFIG isn't set, let's see what we can do with HOME.");
```

Comment:
I don't think this is needed, but we can leave it with log level DEBUG.

Reply:
trace

Reply:
this should also be resolved